curriculum/challenges/english/25-front-end-development/quiz-error-handling/67f412eaf6e68343aab8ea81.md

```diff
@@ -17,439 +17,225 @@ To pass the quiz, you must correctly answer at least 18 of the 20 questions belo
-
-Placeholder distractor 1
-
----
-
-Placeholder distractor 2
-
----
-
-Placeholder distractor 3
-
-#### --answer--
-
-Placeholder answer
-
-### --question--
-
-#### --text--
-
-Placeholder question
+  ```py
+try:
+    print(22 / 0)
+except ZeroDivisionError:
+    print("You can't divide by zero!")
+  ```
 
 #### --distractors--
 
-Placeholder distractor 1
+22
 
 ---
 
-Placeholder distractor 2
+10
 
 ---
 
-Placeholder distractor 3
+0
 
 #### --answer--
 
-Placeholder answer
+You can't divide by zero!
 
 ### --question--
 
 #### --text--
 
-Placeholder question
+Which module of the Python standard library lets you debug your code  in an interractive way?
 
 #### --distractors--
 
-Placeholder distractor 1
+ABC
 
 ---
 
-Placeholder distractor 2
+numpy
 
 ---
 
-Placeholder distractor 3
+debugpy
 
 #### --answer--
 
-Placeholder answer
+pdb
 
 ### --question--
 
 #### --text--
 
-Placeholder question
+Which exception does Python raise when you try to use a method or property that does not exist for that type?
 
 #### --distractors--
 
-Placeholder distractor 1
+`SyntaxError`
 
 ---
 
-Placeholder distractor 2
+`AttributeBug`
 
 ---
 
-Placeholder distractor 3
+`SyntaxBug`
 
 #### --answer--
 
-Placeholder answer
+`AttributeError`
 
 ### --question--
 
 #### --text--
 
-Placeholder question
+Which Python statement can you insert around various points in your code so you can see the values of variables while debugging?
 
 #### --distractors--
 
-Placeholder distractor 1
+`len()`
 
 ---
 
-Placeholder distractor 2
+`console()`
 
 ---
 
-Placeholder distractor 3
+`log()`
 
 #### --answer--
 
-Placeholder answer
+`print()`
 
 ### --question--
 
 #### --text--
 
-Placeholder question
+Which module of the Python standard library lets you write unit tests?
 
 #### --distractors--
 
-Placeholder distractor 1
+Jest
 
 ---
 
-Placeholder distractor 2
+Testpy
 
 ---
 
-Placeholder distractor 3
+UnitPy
 
 #### --answer--
 
-Placeholder answer
+unittest
 
 ### --question--
 
 #### --text--
 
-Placeholder question
+Which error will the code `print("Hello world"` raise in your Python code?
 
 #### --distractors--
 
-Placeholder distractor 1
+It would not raise any error.
 
 ---
 
-Placeholder distractor 2
+`ValueError`
 
 ---
 
-Placeholder distractor 3
+`NameError`
 
 #### --answer--
 
-Placeholder answer
+`SyntaxError`
 
 ### --question--
 
 #### --text--
 
-Placeholder question
+What does `try...except` lets you do in Python?
 
 #### --distractors--
 
-Placeholder distractor 1
+It provides a way to test your code interactively.
 
 ---
 
-Placeholder distractor 2
+It lets you write mathematical expressions.
 
 ---
 
-Placeholder distractor 3
+It speeds up testing.
 
 #### --answer--
 
-Placeholder answer
+It lets you execute a block of code that might raise an exception.
 
 ### --question--
 
 #### --text--
 
-Placeholder question
+Which object lets you access the exception itself for better debugging and direct printing of the error message?
```

Comment:
I think two answers could be interpreted as correct for this question - `Error` and `Exception`.

Most of the built-in _exceptions_ have literally _Error_ in the class name. At the same time most exceptions are subclasses of the `Exception` class, but not all of them. https://docs.python.org/3/library/exceptions.html#exception-hierarchy